doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -54,6 +54,11 @@ Matter
 
 * Added ``EXPERIMENTAL`` select in Kconfig that informs that Matter support is experimental.
 
+Thread
+------
+
+* Added support for Thread 1.2, refer to :ref:`thread_ug_thread_specification_options` and :ref:`thread_ug_feature_sets` for more information.
```

Comment:
I'm not quite happy with this wording since we already had Thread 1.2 support in 1.6.0.

I believe the emphasis should be made in how the way Thread 1.2 support is enabled has changed.

Reply:
fixed